# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Marcel Telka <marcel@telka.sk>, 2004, 2005.
# Mike Karas <zoliqe@gmail.com>, 2007.
# Pavol Šimo <palo.simo@gmail.com>, 2008, 2011.
# Ondrej Šulek <feonsu@gmail.com>, 2008, 2011.
# feonsu <feonsu@gmail.com>, 2011.
# Dimitris Glezos <glezos@indifex.com>, 2011.
msgid ""
msgstr ""
"Project-Id-Version: Firstboot\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-12-03 12:09+0100\n"
"PO-Revision-Date: 2011-06-02 09:30+0000\n"
"Last-Translator: palos <palo.simo@gmail.com>\n"
"Language-Team: Slovak (http://www.transifex.net/projects/p/fedora/team/sk/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: sk\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2\n"

#: ../progs/firstboot:140
msgid "You must be root to run firstboot."
msgstr "Len správca môže spustiť firstboot."

#: ../progs/firstboot:194 ../progs/firstboot:195
msgid "No firstboot modules were found."
msgstr "Nenašli sa žiadne moduly pre firstboot."

#: ../progs/firstboot:205 ../progs/firstboot:206
msgid "Could not create any firstboot interface."
msgstr "Nepodarilo sa vytvoriť žiadne rozhranie pre firstboot."

#: ../firstboot/moduleset.py:92
#, python-format
msgid "Module %s did not set up its UI; removing."
msgstr "Modul %s nenastavil svoje rozhranie; odstránený."

#: ../firstboot/interface.py:79
msgid "Attempted to go back, but history is empty."
msgstr "Pokus o krok späť, ale história je prázdna."

#. If we were previously on the last page, we need to set the Next
#. button's label back to normal.
#: ../firstboot/interface.py:87 ../firstboot/interface.py:163
msgid "_Finish"
msgstr "_Dokončiť"

#: ../firstboot/interface.py:184
msgid "The system must now reboot for some of your selections to take effect."
msgstr ""
"Aby sa prejavili niektoré z vašich volieb, systém musí byť reštartovaný."

#: ../firstboot/interface.py:243
msgid "_Back"
msgstr "_Späť"

#: ../firstboot/interface.py:250
msgid "_Forward"
msgstr "Ď_alej"

#: ../firstboot/interface.py:277
#, python-format
msgid "Module %s did not set up its UI, removing."
msgstr "Modul %s nenastavil svoje rozhranie, odstránený."

#: ../firstboot/interface.py:347 ../firstboot/interface.py:348
msgid "moveToPage must be given a module title or page number."
msgstr "moveToPage musí dostať nadpis modulu alebo číslo stránky."

#: ../firstboot/interface.py:432
msgid "Unable to create the screenshot dir; skipping."
msgstr "Nepodarilo sa vytvoriť adresár snímok obrazovky; vynechané."

#: ../firstboot/interface.py:472 ../firstboot/interface.py:473
#, python-format
msgid "No module exists with the title %s."
msgstr "Modul s nadpisom %s neexistuje."

#: ../firstboot/loader.py:87
#, python-format
msgid "Skipping old module %s that has not been updated."
msgstr "Starý neaktualizovaný modul %s bude preskočený."

#: ../firstboot/loader.py:89 ../firstboot/loader.py:93
#, python-format
msgid ""
"Error loading module %(module)s:\n"
"%(error)s"
msgstr ""
"Chyba pri nahrávaní modulu %(module)s:\n"
"%(error)s"

#: ../firstboot/loader.py:103
#, python-format
msgid "Module %s does not contain a class named moduleClass; skipping."
msgstr "Modul %s neobsahuje triedu s názvom moduleClass; vynechaný."

#: ../firstboot/loader.py:114
#, python-format
msgid ""
"Module %(module)s does not contain the required attribute %(attr)s; "
"skipping."
msgstr "Modul %(module)s neobsahuje požadovaný atribút %(attr)s; vynechaný."

#: ../firstboot/pwcheck.py:108 ../firstboot/pwcheck.py:109
msgid "Very weak"
msgstr "Veľmi slabé"

#: ../firstboot/pwcheck.py:110 ../firstboot/pwcheck.py:111
msgid "Weak"
msgstr "Slabé"

#: ../firstboot/pwcheck.py:112
msgid "Fairly strong"
msgstr "Dosť silné"

#: ../firstboot/pwcheck.py:113
msgid "Strong"
msgstr "Silné"

#: ../firstboot/pwcheck.py:114 ../firstboot/pwcheck.py:115
msgid "Very strong"
msgstr "Veľmi silné"

#: ../firstboot/pwcheck.py:141
msgid "Undefined"
msgstr "Nedefinované"

#: ../modules/date.py:39 ../modules/date.py:40
msgid "Date and Time"
msgstr "Dátum a čas"

#: ../modules/date.py:61
msgid "Please set the date and time for the system."
msgstr "Prosím, nastavte pre tento systém dátum a čas."

#: ../modules/create_user.py:43 ../modules/create_user.py:44
msgid "Create User"
msgstr "Vytvoriť používateľa"

#: ../modules/create_user.py:85
msgid "You must create a user account for this system."
msgstr "Pre tento systém si musíte vytvoriť používateľský účet."

#: ../modules/create_user.py:96
msgid "You must enter and confirm a password for this user."
msgstr "Pre tohto používateľa musíte zadať a potvrdiť heslo."

#: ../modules/create_user.py:103
msgid "The passwords do not match.  Please enter the password again."
msgstr "Heslá sa nezhodujú.  Prosím, zadajte heslo znova."

#: ../modules/create_user.py:115
#, python-format
msgid ""
"The username '%s' is a reserved system account.  Please specify another "
"username."
msgstr ""
"Používateľské meno '%s' je rezervované pre systémový účet. Prosím, zadajte "
"iné používateľské meno."

#: ../modules/create_user.py:134
#, python-format
msgid ""
"A home directory for user %s already exists. Would you like to continue, "
"making the new user the owner of this directory and all its contents?  Doing"
" so may take a while to reset permissions and any SELinux labels.  Would you"
" like to reuse this home directory?  If not, please choose a different "
"username."
msgstr ""
"Domovský priečinok používateľa %s už existuje. Želáte si pokračovať s tým, "
"že sa nový používateľ stane vlastníkom tohto priečinka s celým jeho obsahom?"
" Toto môže chvíľu trvať, kým sa opätovne nastavia práva a prípadné SELinux "
"značky. Želáte si tento domovský priečinok znovu použiť? Ak nie, zvoľte, "
"prosím, iné používateľské meno."

#: ../modules/create_user.py:177
#, python-format
msgid ""
"Fixing attributes on the home directory for %s.  This may take a few "
"minutes."
msgstr ""
"Opravujú sa atribúty domovského priečinka pre %s.  Môže to pár minút trvať."

#: ../modules/create_user.py:197
#, python-format
msgid ""
"Problems were encountered fixing the attributes on some files in the home "
"directory for %(user)s.  Please refer to %(path)s for which files caused the"
" errors."
msgstr ""
"Vyskytli sa problémy s opravou atribútov u niektorých súboroch v domovskom "
"adresári %(user)s.  Prosím, skontrolujte adresár %(path)s na zistenie, ktoré"
" súbory spôsobili chyby."

#: ../modules/create_user.py:223
msgid ""
"You must create a 'username' for regular (non-administrative) use of your "
"system.  To create a system 'username', please provide the information "
"requested below."
msgstr ""
"Je odporúčané vytvoriť 'používateľské meno' pre bežné (nie administrátorské)"
" použitie vášho systému. Na vytvorenie 'používateľského mena' v systéme "
"vyplňte, prosím, nižšie požadované informácie."

#: ../modules/create_user.py:263
msgid "Full Nam_e:"
msgstr "Celé _meno:"

#: ../modules/create_user.py:270
msgid "_Username:"
msgstr "Po_užívateľské meno:"

#: ../modules/create_user.py:277
msgid "_Password:"
msgstr "_Heslo:"

#: ../modules/create_user.py:284
msgid "Confir_m Password:"
msgstr "P_otvrdenie hesla:"

#: ../modules/create_user.py:294
msgid "Add to Administrators group"
msgstr "Pridať do skupiny administrátorov"

#: ../modules/create_user.py:300
msgid ""
"If you need to use network authentication, such as Kerberos or NIS, please "
"click the Use Network Login button."
msgstr ""
"Ak chcete použiť sieťové overovanie totožnosti, ako Kerberos alebo NIS, "
"kliknite, prosím, na tlačidlo Použiť sieťové prihlásenie."

#: ../modules/create_user.py:309
msgid "Use Network _Login..."
msgstr "Použiť _sieťové prihlásenie..."

#: ../modules/create_user.py:317
msgid ""
"If you need more control when creating the user (specifying home directory, "
"and/or UID), please click the Advanced button."
msgstr ""
"Ak potrebujete upraviť ďalšie špecifické nastavenia (domovský adresár, UID),"
" kliknite na tlačidlo Pokročilé."

#: ../modules/create_user.py:327
msgid "_Advanced..."
msgstr "_Pokročilé..."

#: ../modules/create_user.py:365
msgid "Please wait"
msgstr "Čakajte, prosím"

#: ../modules/additional_cds.py:43 ../modules/additional_cds.py:44
msgid "Additional CDs"
msgstr "Doplnkové CD"

#: ../modules/additional_cds.py:55
msgid ""
"Please insert the disc labeled \"Red Hat Enterprise Linux Extras\" to allow "
"for installation of third-party plug-ins and applications.  You may also "
"insert the Documentation disc, or other Red Hat-provided discs to install "
"additional software at this time."
msgstr ""
"Prosím, vložte disk označený \"Red Hat Enterprise Linux Extras\", aby ste "
"umožnili inštaláciu zásuvných modulov a aplikácií od tretích dodávateľov. "
"Tiež môžete vložiť disk s Dokumentáciou (Documentation), alebo ostatné "
"disky, ktoré vám poskytol Red Hat, aby ste si teraz nainštalovali prídavný "
"softvér."

#: ../modules/additional_cds.py:61
msgid "Please insert any additional software install cds at this time."
msgstr "Prosím, vložte teraz prípadné doplnkové inštalačné CD so softvérom."

#: ../modules/additional_cds.py:65
msgid ""
"\n"
"\n"
"To enable runtime support of 32-bit applications on the Intel Itanium2 architecture you must install the Intel Execution Layer package from the Extras disc now."
msgstr ""
"\n"
"\n"
"Na povolenie podpory behu 32-bitových aplikácií na architektúre Intel Itanium2 teraz musíte nainštalovať balík Intel Execution Layer z disku Extras."

#: ../modules/additional_cds.py:73
msgid "Install..."
msgstr "Inštalovať..."

#: ../modules/additional_cds.py:110
msgid ""
"A CD-ROM has not been detected.  Please insert a CD-ROM in the drive and "
"click \"OK\" to continue."
msgstr ""
"CD-ROM nebol nájdený. Prosím vložte CD-ROM do mechaniky a na pokračovanie "
"kliknite na \"Ok\" "

#: ../modules/additional_cds.py:146
msgid "The autorun program cannot be found on the CD. Click \"OK\" to continue."
msgstr "Program autorun nebol nájdený na CD. Na pokračovanie kliknite na \"Ok\"."

#: ../modules/eula.py:35 ../modules/eula.py:36
msgid "License Information"
msgstr "Informácie o licencii"

#: ../modules/eula.py:45
msgid ""
"Thank you for installing Fedora.  Fedora is a compilation of software packages, each under its own license.  The compilation is made available under the GNU General Public License version 2.  There are no restrictions on using, copying, or modifying this code.  However, there are restrictions and obligations that apply to the redistribution of the code, either in its original or a modified form.  Among other things, those restrictions/obligations pertain to the licensing of the redistribution, trademark rights, and export control.\n"
"\n"
"If you would like to understand what those restrictions are, please visit http://fedoraproject.org/wiki/Legal/Licenses/LicenseAgreement."
msgstr ""
"Ďakujeme, že ste si nainštalovali Fedoru.  Fedora je zostavou softvéru, každý so svojou vlastnou licenciou.  Zostava je k dispozícii pod GNU General Public License verzie 2.  Neexistujú žiadne obmedzenia na používanie, kopírovanie alebo modifikáciu tohto kódu.  Aj keď tu sú reštrikcie a záväzky, ktoré sa týkajú redistribúcie kódu, tak v originálnej ako aj v modifikovanej forme.  Okrem iného sa tieto reštrikcie/záväzky týkajú licencovania redistribúcie, ochranných známok a exportných obmedzení.\n"
"\n"
"Ak chcete týmto obmedzeniam dobre porozumieť, prosím navštívte http://fedoraproject.org/wiki/Legal/Licenses/LicenseAgreement."

#: ../modules/eula.py:60
msgid "Understood, please proceed."
msgstr "Rozumiem, prosím pokračovať."

#: ../modules/welcome.py:35 ../modules/welcome.py:36
msgid "Welcome"
msgstr "Vitajte"

#: ../modules/welcome.py:45
msgid ""
"There are a few more steps to take before your system is ready to use.  The "
"Setup Agent will now guide you through some basic configuration.  Please "
"click the \"Forward\" button in the lower right corner to continue"
msgstr ""
"Je potrebné urobiť ešte zopár krokov pred tým, ako bude váš systém "
"pripravený na použitie. Agent nastavenia vás teraz bude sprevádzať základnou"
" konfiguráciou. Na pokračovanie kliknite, prosím, na tlačidlo \"Ďalej\" v "
"pravom dolnom rohu."


